src/Tests/Hooks/BeforeScenarioHooks.cs

```diff
@@ -0,0 +1,27 @@
+﻿using SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure.AzureDurableFunctions;
+using SFA.DAS.Funding.SystemAcceptanceTests.Infrastructure.Configuration;
+using Helper = SFA.DAS.Funding.SystemAcceptanceTests.Helpers.Helper;
+
+namespace SFA.DAS.Funding.SystemAcceptanceTests.Hooks
+{
+    [Binding]
+    public class BeforeScenarioHooks
+    {
+        [BeforeScenario(Order = 1)]
+        public void SetUpHelpers(ScenarioContext context)
+        {
+            var config = Configurator.GetConfiguration();
+            PrintConfig(config); // TODO: DELETE!!!
```

Comment:
Is this meant to be left in for now?

Reply:
yes, will delete when we have this working in azure pipe-lines